scene/gui/tab_container.cpp

```diff
@@ -876,6 +886,7 @@ void TabContainer::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "current_tab", PROPERTY_HINT_RANGE, "-1,4096,1", PROPERTY_USAGE_EDITOR), "set_current_tab", "get_current_tab");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "tabs_visible"), "set_tabs_visible", "are_tabs_visible");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "drag_to_rearrange_enabled"), "set_drag_to_rearrange_enabled", "get_drag_to_rearrange_enabled");
+  ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_hidden_tabs_for_min_size"), "set_use_hidden_tabs_for_min_size", "get_use_hidden_tabs_for_min_size");
```

Comment:
Indentation issue. (Also further down and in header.)

Reply:
The fix is done but in the wrong commit, in that case it might just be better to squash them I guess.

Reply:
Ah indeed.

Reply:
Oh, you are right.. i fixed indentation in wrong commit. Sorry for that. So, Should i squash them?

Reply:
Yeah squashing them should be fine.

Reply:
Done! sorry about that.